### ISSUE TITLE
Unexpected `swaks --help` failure behaviour when `perl-doc` is missing

### ISSUE THREAD

**Issue by polarathene** [jetmore/swaks#106]
Running the command seems to output the source for `/usr/local/bin/swaks` ([where it was installed](https://github.com/docker-mailserver/docker-mailserver/blob/10882f97f23a1244e39771e9987d9443810ace69/target/scripts/build/packages.sh#L45-L50)):

```console
$ swaks --help

#!/usr/bin/env perl

# use 'swaks --help' to view documentation for this program
#
#      Homepage: http://jetmore.org/john/code/swaks/
#   Online Docs: http://jetmore.org/john/code/swaks/latest/doc/ref.txt
#                http://jetmore.org/john/code/swaks/faq.html
# Announce List: send mail to updates-swaks@jetmore.net
#   Project RSS: http://jetmore.org/john/blog/c/swaks/feed/
#       Twitter: http://www.twitter.com/SwaksSMTP

use strict;
use Pod::Usage;
use File::Spec::Functions qw(splitdir);

binmode(STDOUT);
binmode(STDERR);

$|            = 1;
my($p_name)   = (splitdir($0))[-1] =~ m|^(.+?)(\.pl)?$|;
my $p_version = build_version("20240103.0", '$Id$');
my $p_usage   = "Usage: $p_name [--help|--version] (see --help for details)";
my $p_cp      = <<'EOM';
```

After exiting the pager output of that perl source code for swaks, it'll output the more helpful message:

```
You need to install the perl-doc package to use this program.
```

---

I don't know Perl but I assume this behaviour is triggered here?:

https://github.com/jetmore/swaks/blob/8a05259830e67df2ca2b7864c122b43446e1d907/swaks#L44-L47

Perhaps you can avoid the failure scenario with displaying source code, so that only the message regarding `perl-doc` or similar is output?

**Comment by jetmore** [comment]
I came here to say this sounds like a bug with the docker-mailserver packaging, but it looks like you handled that already.  I agree that this could be made nicer, thanks for reporting

**Comment by polarathene** [comment]
> but it looks like you handled that already. I agree that this could be made nicer, thanks for reporting

Yes, we get most of the `perl` packages for swaks implicitly via other software like Postfix IIRC. It was only yesterday I had the `docker-mailserver` container running and I needed to check something with swaks and tried `--help` for the first time 😅 

So I resolved that issue and thought the `swaks --help` UX could be improved, as that was a little jarring to encounter 🤔
